rsmtool/rsmpredict.py

```diff
@@ -98,63 +98,63 @@ def fast_predict(
     trim : bool
         Whether to trim the predictions. If ``True``, ``trim_min`` and
         ``trim_max`` must be specified or be available as attributes of
-        the modeler object.
+        the ``modeler``.
```

Comment:
If load_from_file() and init() both have the object name capitalized in their docstring, perhaps keep that convention for consistency:
```suggestion
        the ``Modeler``.
```